Sources/Confidence/EventStorage.swift

```diff
@@ -114,7 +116,7 @@ internal class EventStorageImpl: EventStorage {
             self.currentFileHandle = try FileHandle(forWritingTo: currentFile)
         } else {
             // Create a brand new file
-            let fileUrl = folderURL.appendingPathComponent(String(Date().timeIntervalSince1970))
```

Comment:
Curious on why this change?

Reply:
because then we don't need to depend on how precise is the timeInterval if we have batch for each event and each event is happening very fast one after another. that was the thought behind it. but then the risk will be the batched file will be 1234.READY and new one will be 1234 or one file has 2 events batched. which is not bad. so feel free to revert this.